comp/process/runner/runnerimpl/runner_test.go

```diff
@@ -27,15 +28,7 @@ import (
 )
 
 func TestRunnerLifecycle(t *testing.T) {
-	_ = fxutil.Test[runner.Component](t, fx.Options(
-		fx.Supply(core.BundleParams{}),
-
-		Module(),
-		submitterimpl.MockModule(),
-		processcheckimpl.Module(),
-		hostinfoimpl.MockModule(),
-		core.MockBundle(),
-	))
+	_ = createDeps(t)
```

Comment:
Because [fxutil.Test fails](https://github.com/DataDog/datadog-agent/blob/98444271e38fa77e02279a7f49fb67d6d2e555be/pkg/util/fxutil/test.go#L51) when it cannot create the object.
Note: I think the value of the initial test was low.

Reply:
okay thx